src/components/TaskEditSuggestions/TaskEditSuggestions.jsx

```diff
@@ -37,42 +39,45 @@ export const TaskEditSuggestions = () => {
   };
 
   return (
-    <Container>
-      <h1 className="mt-3">Task Edit Suggestions</h1>
-      {/* {isUserPermitted ? <h1>Task Edit Suggestions</h1> : <h1>{userRole} is not permitted to view this</h1>} */}
-      {isLoading && <Loading />}
-      {!isLoading && taskEditSuggestions && (
-        <div style={{ overflowX: 'auto' }}>
-          <Table>
-            <thead>
-              <tr>
-                <th onClick={() => dispatch(toggleDateSuggestedSortDirection())}>
-                  Date Suggested <SortArrow sortDirection={dateSuggestedSortDirection} />
-                </th>
-                <th onClick={() => dispatch(toggleUserSortDirection())}>
-                  User <SortArrow sortDirection={userSortDirection} />
-                </th>
-                <th>Task</th>
-                <th></th>
-              </tr>
-            </thead>
-            <tbody>
-              {taskEditSuggestions.map(taskEditSuggestion => (
-                <TaskEditSuggestionRow
-                  key={taskEditSuggestion._id}
-                  taskEditSuggestion={taskEditSuggestion}
-                  handleToggleTaskEditSuggestionModal={handleToggleTaskEditSuggestionModal}
-                />
-              ))}
-            </tbody>
-          </Table>
-        </div>
-      )}
-      <TaskEditSuggestionsModal
-        isTaskEditSuggestionModalOpen={isTaskEditSuggestionModalOpen}
-        taskEditSuggestion={currentTaskEditSuggestion}
-        handleToggleTaskEditSuggestionModal={handleToggleTaskEditSuggestionModal}
-      />
-    </Container>
+    <div className={darkMode ? 'bg-oxford-blue text-light' : ''} style={{minHeight: "100%"}}>
```

Comment:
```suggestion
     const  darkMode ={
     textDarkMode : darkMode ? 'text-light' : '',
     ......
```
The idea here is put in one place where we can easily find and edit rather than doing in line.